src/lib/components/GroupTree/components/Settings/DateTimeSlider.tsx

```diff
@@ -0,0 +1,79 @@
+import {
```

Comment:
This sort of components are pretty generic and can be useful for many things, so in the future we should probably have them in webviz core components.

Reply:
Yes, I agree too. Most of the UI components are directly using eds or material UI components, but for this DateTime selector, I think it makes sense to move it into the webviz-core-component, especially because we now have other improvements requests (e.g. https://github.com/equinor/webviz-subsurface-components/issues/388), and therefore it might grow into a more complex component.